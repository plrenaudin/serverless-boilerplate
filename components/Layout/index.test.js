import Layout from ".";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <Layout>
        <div>Children content</div>
      </Layout>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
