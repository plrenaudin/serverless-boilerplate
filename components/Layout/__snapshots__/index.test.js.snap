// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <nav>
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="/about"
      onClick={[Function]}
    >
      About
    </a>
  </nav>,
  <main>
    <div>
      Children content
    </div>
  </main>,
]
`;
